src/makeCalculator.js

```diff
@@ -5,6 +5,35 @@
  */
 function makeCalculator() {
   // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
```

Comment:
Instead of using a getter for `result`, you can simply include it as a property within the object. Getters are typically used when you need to compute or format the value before returning it, which is not the case here.